docker-compose.yml

```diff
@@ -57,3 +57,29 @@ services:
     container_name: openzipkin
     ports:
       - 9411:9411
+
+  #to build this image run 'docker build -t slackhq/kaldb .'
+  kaldb:
+    image: 'slackhq/kaldb'
+    container_name: kaldb_kaldb
+    profiles:
+      - kaldb
+    ports:
+      - 8080:8080
+      - 8081:8081
+      - 8082:8082
+      - 8083:8083
+      - 8085:8085
+      - 8086:8086
+    environment:
+      - KALDB_ZK_CONNECTION_STRING=zookeeper:2181
+      - KAFKA_BOOTSTRAP_SERVERS=kafka:29092
+    depends_on:
```

Comment:
I had to change the depends_on to this for it to run locally
```
    depends_on:
      - zookeeper
      - kafka
      - s3
      - openzipkin
```

I think the `depends_on` doesn't exist in docker 3 ? https://stackoverflow.com/questions/47710767/what-is-the-alternative-to-condition-form-of-depends-on-in-docker-compose-versio but then re-introduced in 3.9 https://stackoverflow.com/questions/71060072/docker-compose-depends-on-with-condition-invalid-type-should-be-an-array ?

Reply:
@vthacker - I removed the condition clauses to match what you have above and it still starts up properly locally for me. I've pushed the change up.